src/modules/list.js

```diff
@@ -18,11 +18,24 @@ const tasks = (dataTask) => {
     input.type = 'checkbox';
     input.id = 'checkbox';
     input.className = 'checkbox';
+    input.addEventListener('change', (event) => {
+      const index = task.tasks.findIndex((t) => listItem.index === t.index);
+      if (index === -1) {
+        return;
+      }
+
+      task.tasks[index].completed = event.target.checked;
+    });
     div.appendChild(input);
     const desc = document.createElement('p');
     desc.id = 'list-header';
-    desc.innerText = `${listItem.description} ${listItem.index}`;
+    desc.innerText = `${listItem.description} ${listItem.index + 1}`;
     div.appendChild(desc);
+    if (listItem.completed) {
+      input.setAttribute('checked', 'checked');
+      desc.style.textDecoration = 'line-through';
+      desc.style.color = '#A3A3A3';
+    }
```

Comment:
- [ ] When the user clicks on the checkbox the completed status should be updated immediately and this is not happening in your app. When I click on the checkbox nothing happens to the complete status and the local storage is not changed, preventing the 'Clear all completed' button to work properly.  Kindly fix that issue.
 - [OPTIONAL] It is not good for the visual of the tasks to have the index of the task appear right after the task name, this can also lead the user to a bad user experience when he/she tries to edit the task name.
 - 
<img width="1093" alt="image" src="https://user-images.githubusercontent.com/61919617/184403954-d4d328ce-f9b8-44ef-8659-43569d5e81f3.png">